source/plugin/bookview.rst

```diff
@@ -7,11 +7,10 @@ Book Views
     org.spongepowered.api.item.inventory.ItemStack
     org.spongepowered.api.text.BookView
     org.spongepowered.api.text.BookView.Builder
-    org.spongepowered.api.text.Text
     java.util.Collection
 
 A :javadoc:`BookView` is the representation of the Book GUI on the client. The ``BookView`` is not associated with an
-actual :javadoc:`ItemStack` and is only for displaying :javadoc:`Text` through a book to the player. Note that a
+actual :javadoc:`ItemStack` and is only for displaying ``Component`` through a book to the player. Note that a
```

Comment:
It should be possible to link to the javadoc here?

Reply:
Yes, if the adventure javadocs get added here: https://github.com/SpongePowered/SpongeDocs/blob/stable/source/conf.py#L92

Reply:
Added :)